superduperdb/db/base/db.py

```diff
@@ -513,6 +600,47 @@ def replace_children(r):
             getattr(self, cm)[m.identifier] = m
         return m
 
+    def _build_delete_task_workflow(
+        self, 
+        query: Delete,
+        ids: t.List[str],
+        verbose: bool = False,
+    ):
+        G = TaskWorkflow(self)
+
+        vector_indices = self.show('vector_index')
+
+        if not vector_indices:
+            return G
+
+        deleted_table_or_collection =  query.collection_or_table.identifier 
+
+        for vi in vector_indices:
+            vi = self.vector_indices[vi]
+            listener_table_or_collection = \
+                vi.indexing_listener.select.table_or_collection.identifier
+
+            if deleted_table_or_collection != listener_table_or_collection:
+                continue
+
+            if s.CFG.vector_search == 'in_memory' and vi.identifier not in self.fast_vector_searchers:
```

Comment:
self.fast_vector_searchers what's this?

Reply:
This is the tables of vectors/ connections to `lance` to actually perform the vector-search.

Reply:
I added this to the top level so that CDC is somehow easier.